features/player_stats.lua

```diff
@@ -186,7 +188,7 @@ local function robot_built_entity(event)
     end
 
     -- When item gets built, add to the total entities built for the player whose robot built the entity NOT for the player who placed the ghost
-    local robot_owner = robot.logistic_network.cells[1].owner
+    local robot_owner = event.robot.logistic_network.cells[1].owner
```

Comment:
Probably something that changed when resolving the merge conflict.
```suggestion
    local robot_owner = robot.logistic_network.cells[1].owner
```